src/main/java/seedu/address/model/property/Deadline.java

```diff
@@ -26,6 +32,16 @@ public Deadline(LocalDate deadline) {
         this.deadline = deadline;
     }
 
+    /**
+     * Returns true if a given string is a valid date.
```

Comment:
Valid date or should it be deadline?

Reply:
Ohh, I left it as date because it is checking if it is an actual date. Should I change to deadline?

Reply:
probably a small issue and both are ok but it think `deadline` makes more sense since you are checking for a valid `deadline`?

Reply:
ok shall update this